### ISSUE TITLE
Getting Data from Pool Controller but Unable to Control Equipment from HA

### ISSUE THREAD

**Issue by FlaMike** [Crewski/njsPC-HA#14]
I migrated my pool controller system from running on a Pi to Ubuntu Server docker containers.  The containers spun up and the logs are error-free.  I can control the pool equipment from the poolcontroller-dashboard web interface.

I reloaded the HACS integration and restarted HA.  I  then reloaded the HA integration.

All of the entities in the HA dashboard populated with what appears to be good data and status information.

However, when I try to turn lights/pumps/etc. on and off, the UI switches flip back to their previous positions and the equipment status does not change.  There are no entries in the HA logbook.

At this point, I am not sure what is causing the problem or where to look for errors or other information that may lead me to a solution.  Any help & suggestions are appreciated greatly.

Thank you!

**Comment by Crewski** [comment]
Hmmm....sounds like the socket.io isn't working.  If you make a change via dashPanel, does it change in Home Assistant as well (I'm guessing it doesn't).  Also, if you make a change in Home Assistant, does it change in dashPanel?  All of the actions (on/off, set temp, etc) are set via RESTful API, all the data back is sent via socket.io.

**Comment by FlaMike** [comment]
If, for example, I turn the lights on using the dashPanel, the pool lights *do* go on, but the HA dashboard does not show the change.  Similarly, if I turn the Polaris on from HA's dashboard, neither the equipment nor the dashPanel reflect the HA command.  (I also tried using call service from the developer's tools with the same result.)

What do I need to do to reestablish socket.io?

**Comment by FlaMike** [comment]
OK.  I am a stupid fool.  All my bad.  I did not turn off the Pi most recently when I moved the dongle to the docker server.  So I was still getting communication (data) through the Pi.  But the Pi had no path to communicate with the pool controller (no dongle!).  I turned off the Pi, and magically, all of the data disappeared from HA.  And I got errors in your integration in HA.  So I reloaded the HACS integration, restarted HA, removed & the reinstalled you HA integration & POOF!  It all works again.

As I said, all my bad, not yours!  Sorry to have bothered you with my stupidity.

Have a super night & beyond!!!  Time to retire the Pi :-)
